source/stylesheets/settings/_typography.scss

```diff
@@ -22,6 +22,7 @@ h3 {
 h4 {
   font-size: $font-size-large;
   font-weight: $font-weight-medium;
+  line-height: $line-height-base;
 }
 
```

Comment:
I'm really enjoying that you're abstracting your typography like this. I might even take it a step further and work this into a class instead of applying it to a `h4` element selector, as there might be a situation where you'd want a h4-looking header that uses another heading element. This way you're not sacrificing accessibility for aesthetics!